doc/sphinxman/source/occ.rst

```diff
@@ -374,7 +374,24 @@ Starting in v1.4, MP2.5 and MP3 default to the density-fit algorithm. Set |globa
 
 Publications resulting from the use of the non-OO CC codes should cite the following publications:
 
-* **REMP** [Behnle:2019:REMP]_, [Behnle:2022:OREMP]
+* **MP2** [Bozkaya:2011:omp2]_, [Bozkaya:2013:omp2grad]_, and [Bozkaya:2014:dfomp2grad]_
```

Comment:
I would much rather we separate out the DF and the non-DF implementations. A paper using conventional OMP2 has no business citing the DF-OMP2 paper. That'll just confuse people following citations.